src/xdocs/cmdline.xml.vm

```diff
@@ -334,18 +334,35 @@ java -D&lt;property&gt;=&lt;value&gt;  \
           (Root module, Checks, etc)</a> in configuration file:
         </b>
       </p>
+
+      <p>For Linux/Unix OS:</p>
       <div class="wrap-content">
         <source>
-          java -classpath MyCustom.jar;checkstyle-${projectVersion}-all.jar \
+          java -classpath "MyCustom.jar:checkstyle-${projectVersion}-all.jar" \
           &#xa0;&#xa0;&#xa0;&#xa0;com.puppycrawl.tools.checkstyle.Main -c config.xml Check.java
         </source>
       </div>
+
+      <p>For Windows OS:</p>
+      <div class="wrap-content">
+        <source>
+          java -classpath "MyCustom.jar;checkstyle-${projectVersion}-all.jar" ^
+          &#xa0;&#xa0;&#xa0;&#xa0;com.puppycrawl.tools.checkstyle.Main -c config.xml Check.java
```

Comment:
DOS:
````
M:\>java -classpath "MyCustom.jar;checkstyle-10.17.0-all.jar" ^
More?               com.puppycrawl.tools.checkstyle.Main -c config.xml Check.java
Files to process must be specified, found 0.

````

PowerShell:
````
PS M:\> java -classpath "MyCustom.jar;checkstyle-10.17.0-all.jar" ^
Error: Could not find or load main class ^
Caused by: java.lang.ClassNotFoundException: ^
PS M:\>               com.puppycrawl.tools.checkstyle.Main -c config.xml Check.java
````

I don't feel this current documentation is enough for Windows users. I also find it odd the first thing we would add here would be DOS' instead of PS. On Windows 10, you have to specifically go through the trouble to open DOS. Only option given in Windows Explorer is PS.

Reply:
@mattxwang , please share pictures of how it executed on your side.

Reply:
@mattxwang , ping.